.storybook-s2/docs/Release Notes.mdx

```diff
@@ -4,6 +4,45 @@ export default MDXLayout;
 
 # Release Notes
 
+## v0.5.0
+
+This version includes a major update to the way Dialogs are created. Previously, the Dialog API largely followed S1's Dialog's API
```

Comment:
Oh hmm, maybe we got our wires crossed. I thought this was assigned to me in the ticket, and I wrote a version of this in the canvas. Maybe we can combine them...

Reply:
I thought we had two release notes — one for RSP/RAC and then another one for S2. This PR seems to handle the S2 stuff and the canvas seems to be more for the RSP/RAC stuff?

Reply:
oh doh, I conflated my work as the S2 release notes in general haha. I'm happy to pull yours in

Reply:
@yihuiliao right but the dialog api changes are S2 specific, I made the same assumptions haha hence I didn't really look at the canvas while writing these